Update getting_started.rst

hwpang opened the pull request:
### Description

Fixing the path in getting started examples

### Types of changes
<!--- Put an `x` in all the boxes that apply, and remove the not applicable items -->
- [ ] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [x] Documentation updated.

YuanTingHsieh commented:
hi @hwpang , thanks for the interest, could you provide the motivation as to why we don't want to "mkdir simulator-example"?

hwpang commented:
Hi @YuanTingHsieh , with `mkdir simulator-example`, the copied folder has structure `simulator-example/hello-pt/requirements.txt`, while without it, the copied folder has structure `simulator-example/requirements.txt`. I made these changes to make the folder structure under `simulator-example` consistent throughout the getting started example.

YuanTingHsieh commented on docs/getting_started.rst:
```diff
@@ -295,7 +295,6 @@ to a working directory:
 .. code-block:: shell
 
-  mkdir simulator-example
   cp -rf NVFlare/examples/hello-world/hello-pt simulator-example/
```
@hwpang to ensure the behavior is always consistent, we probably need to add a `rm -rf simulator-example` in front of this command to achieve what we want

hwpang replied:
@YuanTingHsieh That makes sense, thanks! Updated!

YuanTingHsieh reviewed (commented):
suggest one change

YuanTingHsieh reviewed (approved):
Thanks, LGTM

@SYangster and @nvkevlu 

other thing I notice we should remove this "python 3.8" requirement text in getting started doc, we can do that in other PR

YuanTingHsieh commented:
/build

YuanTingHsieh commented:
@hwpang 

Can you think of any benefit we change from "mkdir" to "rm -rf"?

hwpang commented:
@YuanTingHsieh Thanks. I realize that there is a simpler fix than what I implemented before. I have implemented the new fix to add the `hello-pt` in the `python3 -m pip install -r simulator-example/hello-pt/requirements.txt`. This way, we don't need to change any other part of the documentation. I followed through this new instruction locally, and it works for me.

YuanTingHsieh reviewed (approved):
Thanks, LGTM

YuanTingHsieh commented:
/build

SYangster commented:
Hi @hwpang, it seems that your most recent commit (https://github.com/NVIDIA/NVFlare/pull/2737/commits/4200f77b7b04741468d0082cfe10e36f35d2d71c) was not signed. Once it is signed, then it'll be ready to merge, thanks!

hwpang commented:
@SYangster Thanks! I have signed and rebased to main.

SYangster commented:
/build
